pkg/cluster/internal/kubeadm/config.go

```diff
@@ -311,15 +312,19 @@ apiServer:
     "feature-gates": "{{ .FeatureGatesString }}"
 controllerManager:
   extraArgs:
+{{ if .FeatureGates -}}
```

Comment:
move up a few lines so it includes the apiserver feature-gates

Reply:
sigh, this part was correct, moving it up broke the other extraArgs.